lib/check_params.h

```diff
@@ -397,6 +401,22 @@ void printQudaInvertParam(QudaInvertParam *param) {
   P(omega, INVALID_DOUBLE);
 #endif
 
+#ifdef INIT_PARAM
+  P(ca_basis, QUDA_INVALID_BASIS);
+  P(ca_lambda_min, 0.0);
+  P(ca_lambda_max, -1.0);
+#else
+  if (param->inv_type == QUDA_CA_CG_INVERTER ||
+      param->inv_type == QUDA_CA_CGNE_INVERTER ||
+      param->inv_type == QUDA_CA_CGNR_INVERTER) {
```

Comment:
clang-format suggested the following change:


```
-  if (param->inv_type == QUDA_CA_CG_INVERTER ||
-      param->inv_type == QUDA_CA_CGNE_INVERTER ||
-      param->inv_type == QUDA_CA_CGNR_INVERTER) {
+  if (param->inv_type == QUDA_CA_CG_INVERTER || param->inv_type == QUDA_CA_CGNE_INVERTER
+      || param->inv_type == QUDA_CA_CGNR_INVERTER) {
```

<!-- identifier: clang-format -->
<!-- File: lib/check_params.h -->